twilight-model/src/guild/onboarding/prompt_type.rs

```diff
@@ -0,0 +1,10 @@
+use serde::{Deserialize, Serialize};
+
+/// The type of prompt.
+#[derive(Clone, Copy, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub enum OnboardingPromptType {
+    /// A prompt that allows the user to select multiple options.
+    MultipleChoice,
+    /// A prompt that allows the user to use a dropdown to select an option.
+    DropDown,
+}
```

Comment:
Should serialize as integers and it need to handle unknown variants (see other `Type` enums).